lib/model/model.go

```diff
@@ -1243,20 +1243,69 @@ func (m *Model) updateLocals(folder string, fs []protocol.FileInfo) {
 		return
 	}
 	files.Update(protocol.LocalDeviceID, fs)
-
+	
 	filenames := make([]string, len(fs))
 	for i, file := range fs {
 		filenames[i] = file.Name
 	}
-
+	
+	// Lets us know if file/folder change was originated locally or from a network
+	// sync update.  Now write these to a global log file.
+	if remoteUpdate != true {
+		m.writeToGlobalLog(m.folderCfgs[folder].Path(), fs)
+	}
+	
 	events.Default.Log(events.LocalIndexUpdated, map[string]interface{}{
-		"folder":    folder,
-		"items":     len(fs),
-		"filenames": filenames,
-		"version":   files.LocalVersion(protocol.LocalDeviceID),
+		"folder":       folder,
+		"items":        len(fs),
+		"filenames":    filenames,
+		"version":      files.LocalVersion(protocol.LocalDeviceID),
+		"remoteupdate": remoteUpdate,	
```

Comment:
I'd call this simply `remote` as it's obvious it's a IndexUpdate from the event type.